Swiftification/Dictionary.swift

```diff
@@ -56,6 +56,17 @@ public extension Dictionary {
         return result
     }
     
+    /// Unions the other dictionaries with `self`. If more than
+    /// one dictionary has the same key, it will take on the rightmost
+    /// dictionary's value.
+    mutating func formUnion(first: Dictionary, _ rest: Dictionary...) {
```

Comment:
The code looks okay, but is `formUnion` the best name for this function? It seems like `combine` or just `union` would be better. Why did you choose `formUnion`?